WalletWasabi/Models/ErrorDescriptor.cs

```diff
@@ -17,5 +17,10 @@ public bool Equals(ErrorDescriptor other)
 		{
 			return (Severity == other.Severity && Message == other.Message);
 		}
+
+		public override string ToString()
```

Comment:
why do we need to do this?

Reply:
It was used when we did not override ErrorDataTemplate yet. I removed this.